### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by Riley31415** [Riley31415/InfinityExpansion#28] [Ideas]
post suggestions and ideas here

**Comment by Fharen** [comment]
I've been checking and there's currently no machine way to farm slimes. 
would be great if we can add Slime in mob simulation.

Thanks!

**Reply by Riley31415** [reply]
done 👍

**Comment by Fharen** [comment]
Infinity Cargo manager:
what it does:
place/pull 2 stacks (128 items), instead of the original 1 stack (64 items)

reason:
current cargo can only pull 1 stack per tick,. this causes machines like Quarries buffer chests to clog.
since current cargo system can only pull 1 stack at the first slot per tick at a time, if there's only 4 items on the first slot, it will only pull 4 items on that tick and wait for another tick for the next transfer. but once it pulls an item, a new low stack will appear in the first slot and the next slots will accumulate slowly and eventually clog the system.

**Reply by Riley31415** [reply]
Not gonna happen due to the large amount of code required and cargo being rewritten soon tm

**Comment by Vexiusnetwork** [comment]
Possible additions:
1.Needs slime and ghasts adding to Mob Simulation Cards
2.Needs a way to generate Sulphates without having to mine mountains of netherack.  You could add eg ghast tears and sulphates to the quarry output, or make another machine that only works in nether and mines  sulphates, ghast tears, bone, quartz, redstone, netherite and maybe essence of afterlife or necrotic skulls.

**Reply by Riley31415** [reply]
you can automate netherrack with quarries

**Comment by RelativoBR** [comment]
Include cost xp to unlock.

_import io.github.thebusybiscuit.slimefun4.core.researching.Research;_

Reason: All items already start unlocked, we would like to add some difficulty in releasing the infinity items.

**Reply by Riley31415** [reply]
Added in build 118, enable in config

**Comment by anonisnap** [comment]
A Config File or something, allowing for colour changes. Specifically for the Signs attached to Storage Units.

Reason: The different coloured signs don't all work well with the Text on it
![2021-07-31_02 39 48](https://user-images.githubusercontent.com/42655737/127723756-038abd3a-861f-4c1c-a8fc-e56913c8f594.png)

**Reply by Riley31415** [reply]
Would white be better?

**Reply by anonisnap** [reply]
Before today it was White. Looks a lot better on most of the signs, A little unsure of how it'll look on Birch signs, seeing as they are fairly light in their colouring already

**Reply by Riley31415** [reply]
Build 118 will make it so anything with no color becomes white, hopefully that works well

**Comment by anonisnap** [comment]
Allow Singularity Constructors to use the blocks of a material, eg: Redstone Block, to add 9 to the Redstone Singularity.
Though it might cause trouble if a constructor is missing 1 Redstone Dust, and you put in 1 Redstone Block.

**Comment by eggveloper** [comment]
update to 1.18.1

**Comment by EletricDragonYT** [comment]
Expand Mob Simulations, the mob simulation is missing several mobs such as rabbits and strays both of which have unique drops. Another thing that could be expanded on is the drops from mob simulations, something that would be nice is drops which are related to the mob as well as a chance to obtain that drop. An example of this is and:

Husks
- Rotten Flesh (2-5)
- Sand (1-3 | 5% chance)
- Husk Head (2.5% chance)

Rabbits
- Raw Rabbit (1-2)
- Rabbit Hide (2-4)
- Rabbits Foot (5% chance)
- Rabbit Head (2.5% chance)

Snowman
- Snowballs (1-4)
- Block of Snow (50% chance)
- Ice (2-6 | 50% chance)
- Packed Ice (1-4 | 25% chance)
- Blue Ice (10% chance)
- Snowman Head (2.5% chance)
- Pumpkin-less Snowman Head (1% chance)

Another potential addition is the ability to grant the simulation enchantments like Looting, Sharpness, and Fire Aspect, this would allow players to get more for less simulations and would allow players to obtain loot faster as well as cooked over raw.

**Comment by Vix2250** [comment]
I love Slimefun and Infinity so thanks so much for all the work you are putting in it! Our server just updated our SF and we got Networks. So decided to rebuild my factory using Networks instead of cargo and it works so well. I'm a automation manic and love automating things. One thing I notice I would really love is a small Infinity Storage unit. One similar to a single chest or 1728 items. When you automate say Blistering Ingots 6400 items is just a tad to much especially if you have to wait for all of them to fill up before you start getting in Carbonado
